spk/pyload/src/app/config

```diff
@@ -0,0 +1,12 @@
+{
+    ".url": {
+        "com.synocommunity.packages.pyload": {
+            "title": "pyLoad",
+            "desc": "pyLoad",
+            "icon": "images/pyload-{0}.png",
+            "type": "url",
+            "url": "3rdparty/pyload/pyload.cgi",
```

Comment:
OK, I set it to static on http. Everyone who want to use it with SSL has to set everything manually.